### PR TITLE
Details page

### LINKED ISSUE #22: Build Details Page
Components needed:
navigation 2
search bar
header 2
3 column recipe
recipe direction

### PR DESCRIPTION
**Pull request by kststudio**
fixes #22

### CODE REVIEW

File: frontend/src/components/details/Gallery.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
```
Comment (sdymj84) [inline frontend/src/components/details/Gallery.js]: Component is unused

**Review by sdymj84 — CHANGES_REQUESTED** [review]
image

Since DetailsHero and MainHero are pretty identical except for title and content, I would create one 'Hero.js' file under the global folder and invoke it from main and detail page as below.

Hero.js
image

main.js
image

details.js
image


so that we make Hero.js reusable component and use it anywhere it needs to be. I would say this is the whole point of why we use component-based frameworks

File: frontend/src/components/details/Gallery.js (hunk 1 above)
Comment (kststudio) [reply frontend/src/components/details/Gallery.js]: this is all set

File: frontend/src/components/details/RecipeInstructions.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const RecipeInstructions = () => {
+    return (
+        <div class="container">
```
Comment (sdymj84) [inline frontend/src/components/details/RecipeInstructions.js:5]: error from 'class'
I'll merge and fix it from my PR

**Review by sdymj84 — APPROVED** [review]
RecipeInstructions.js has warning with 'class'
I'll merge and fix on my PR
Thank you!

**Pull request merged**